src/create_disk.sh

```diff
@@ -119,6 +125,31 @@ do
 done
 ostree admin deploy "${deploy_ref}" --sysroot rootfs --os "$os_name" $kargsargs
 
+# This will allow us to track the version that an install
+# originally used; if we later need to understand something
+# like "exactly what mkfs.xfs version was used" we can do
+# that via looking at the upstream build and finding the
+# build logs for it, getting the coreos-assembler version,
+# and getting the `rpm -qa` from that.
```

Comment:
> finding the build logs for it, getting the coreos-assembler version, and getting the `rpm -qa` from that.

Is that true though? Most likely except for recent builds, the cosa image would have long been GC'ed already. We should probably include a `buildroot.txt` file or something in the build dir listing all the packages in the container.

Another approach is building cosa using lockfiles (and storing them in this repo), which has nice properties though I don't think it's worth the overhead right now.